MEMBER_EXPECTATIONS.md

```diff
@@ -1,31 +1,58 @@
+* Be the model of accountability and leadership. Provide the example of ownership and stewardship that everyone can follow to success.
+* Commit to ongoing development and governance best practices. Contribute good ideas, successful policies, better workflows, and other lessons learned back to the Working Group.
+* Follow the contribution guidelines of the organization or project to which you are contributing.
 
-If a representative has an objection or dissent they should express it as early as possible to ensure
-there is ample time to discuss and reach consensus before representing Standards Working Group.
+### Building Consensus
+Representatives may belong to one or more projects in the Foundation, and as such should feel free to discuss topics with each relevant group. In the case of conflicting opinions between Foundation projects, the Representative may raise an issue to the Standards Working Group for support. 
 
-Representatives must also conduct themselves in a professional and respectful manner. Some general guidelines include:
+Representatives or Working Group Members may also call for an ad-hoc meeting to discuss controversial topics, or topics wherein a general consensus view has not yet emerged. Representatives are strongly encouraged to schedule ad-hoc meetings to refine thinking and inform stakeholders prior to plenary or liaison meetings where they will be discussed. Representatives should focus on developing and sharing the consenus of the Foundation and the projects impacted by the topic at hand. 
 
-* When in doubt/foreseeing any potential conflict, a representative should choose one "role" with which to express an opinion instead of invoking multiple at once.
-* Aim to remediate first and then discuss. If other members of the group express concerns about actions, acknowledge their concerns by stopping the actions in question and then discuss within the group to come to a common agreement.
-* Treat all community members with respect, consideration, and highest standards of ethical conduct.
-* Build trust by keeping your promises.
-* Be the model of accountability and leadership. Provide the example of ownership and stewardship that everyone can follow to success.
-* Commit to ongoing development and learning best practices for governing.
-* Follow the code of conduct of both the Open JS Foundation, _and_ the organization they are a representative to.
+### Representatives and Decision-making
+Some of OpenJS Foundation's memberships and liaison relationships permit Representatives to participate in their formal decision-making processes (e.g. voting). Representatives should disclose any vote to the Standards Working Group, and provide a recommendation. If the Representative has an objection or plans to cast a dissenting vote, they should express it as early as possible to ensure there is ample time to discuss and reach consensus on the intended approach.
```

Comment:
```suggestion
Some of OpenJS Foundation's memberships and liaison relationships permit Representatives to participate in their formal decision-making processes (e.g. voting). Representatives should disclose any vote to the Standards Working Group, and provide a recommendation. If the Representative has an objection or plans to cast a dissenting vote while representing the Foundation, they should express it as early as possible to ensure there is ample time to discuss and reach consensus on the intended approach.
```

Reply:
That a good step forward. I'd be a little more prescriptive:

```suggestion
Some of OpenJS Foundation's memberships and liaison relationships permit Representatives to participate in their formal decision-making processes (e.g. voting). Representatives should update the Standards Working Group of such opportunities that are relevant to the interests of OpenJSF or its projects well ahead of time so that the group is able to each consensus on the intended approach. The Representative must represent the group's decision and abstain in case the group hasn't been able to reach consensus.
```

I feel that this covers dissent (as the WG won't reach consensus in case the Rep disagrees with the majority).

Reply:
I'm a bit confused.

While wearing the Foundation hat, we've already discussed and decided on how objections work - brought to Standards beforehand, and if it can't be for any reason, best judgement can be used and may need defending at the next Standards meeting.

While not representing the Foundation, an individual is under no obligation to interact with the Standards group about objections, even if they're a member of the Standards group.

Reply:
I _think_ I understand the source of confusion.

Following some of the discussion and proposed changes in https://github.com/openjs-foundation/standards/pull/239, I was using the term _Representative_ in a much narrower way, i.e. to specifically mean when a person is representing OpenJSF (e.g. "@jorydotcom is OpenJSF's W3C _Representative_") but not when a person is operating in a working group in more of an _Invited Expert_ capacity.

Would it make sense to distinguish these roles more precisely? I think @rginn used the term _subject matter expert_ (SME) in a related issue which is more common than _Invited Expert_. Would that work for folks?

Reply:
A person might be operating as a delegate of their employer while still being associated with OpenJS, and as such they don't need to answer to the Standards WG, so it's got nothing to do with IE/SME status imo.

Reply:
> While wearing the Foundation hat, we've already discussed and decided on how objections work - brought to Standards beforehand, and if it can't be for any reason, best judgement can be used and may need defending at the next Standards meeting.

Sorry, I didn't see this documented anywhere. To be clear, I much prefer this as a solution. I think trusting that people will do the right thing and be proactive about building consensus if the topic feels more contentious is the right thing to do. However, I don't think it is acceptable for a _Representative_ of the OpenJSF in the narrow sense (i.e. not an _SME_), to vote against the consensus of the WG (and if the WG can't get to consensus, the Rep should abstain).

> While not representing the Foundation, an individual is under no obligation to interact with the Standards group about objections, even if they're a member of the Standards group.

100%. We should also make very clear that the persons is acting in an SME capacity in that case and not representing OpenJSF.

Reply:
See #84 for previous discussion.

I completely agree with your two roles, and I think that's basically what #84 came to decide :-) we can bikeshed the terms later.

Reply:
Awesome. I wasn't aware of that thread. Thanks for sharing. I don't really care what the terms are either. It would just be good if we could avoid them creating more confusion by overlapping with common terms already used by SDOs (e.g. delegate which means completely different things for Ecma and OSI).

Reply:
it does seem like we need a mapping table 😄